src/items.ts

```diff
@@ -1,124 +1,1166 @@
+/**
+ * items.ts
+ * All available items and their properties
+ * lgfrbcsgo & Nikolaus - October 2020
+ */
+
 export type Liter = number
 export type Minutes = number
 export type Quantity = number
 
-export type Item = Machine | Container | Ore | Other
+export type Item = Ore | Catalyst | Product | Element
```

Comment:
What's the difference between a `Product` and an `Element` semantically? Could we combine those into one category `Other`?
- `Ore` can't be produced. 
- `Catalyst`s may not need to be produced. 
- `Other` needs to be produced. 

`Other` could be its own union type if we need more granularity. E.g.
```typescript
type Other = Element | Product | Container | ...
```

Reply:
I was thinking: `Element` can't be an ingredient

Reply:
I haven't thought about that. Great idea.